iOS/Settings/Appearance/ArticleThemeManagerView.swift

```diff
@@ -0,0 +1,182 @@
+//
+//  ArticleThemeManagerView.swift
+//  NetNewsWire-iOS
+//
+//  Created by Stuart Breckenridge on 20/12/2022.
+//  Copyright © 2022 Ranchero Software. All rights reserved.
+//
+
+import SwiftUI
+
+struct ArticleThemeManagerView: View {
+    
+	@StateObject private var themeManager = ArticleThemesManager.shared
+	@State private var showDeleteConfirmation: (String, Bool) = ("", false)
+	@State private var showImportThemeView: Bool = false
+	@State private var showImportConfirmationAlert: (ArticleTheme?, Bool) = (nil, false)
+	@State private var showImportErrorAlert: (Error?, Bool) = (nil, false)
+	@State private var showImportSuccessAlert: Bool = false
+	
+	var body: some View {
+		Form {
+			Section(header: Text("Built-in Themes", comment: "Section header for installed themes"), footer: Text("These themes cannot be deleted.", comment: "Section footer for installed themes.")) {
```

Comment:
This `Section(header:` call is [deprecated in iOS 16.4](https://developer.apple.com/documentation/swiftui/section/init(header:content:))

They suggest to use [`Section(content:, header:)`](https://developer.apple.com/documentation/swiftui/section/init(content:header:)-29e2l) instead